OrbitBase/CMakeLists.txt

```diff
@@ -35,6 +35,16 @@ target_sources(OrbitBase PRIVATE
         Tracing.cpp
         SafeStrerror.cpp)
 
+if(WIN32)
+target_sources(OrbitBase PRIVATE
+        include/OrbitBase/Platform/Windows/Profiling.h
```

Comment:
Is the "intermediate" `Platform` directory necessary? It seems that it just makes the path longer but without real need.